src/Core/Helpers/AbstractConfigHelper.php

```diff
@@ -53,6 +59,8 @@ abstract class AbstractConfigHelper {
   abstract public function getStockBufferValue();
 
   /**
+   * Retrieve the configuration values for setting up a test environment
```

Comment:
This checks if the plugin is configured to use the **test** / dryRun mode when communicating with the Wayfair APIs.
I don't know what you mean about test environments.